app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java

```diff
@@ -116,6 +121,9 @@ public void onResume() {
     @Override
     public void onStop() {
         super.onStop();
+        int position = binding.recyclerView.getFirstVisiblePosition();
+        Log.d(TAG, "onStop: " + position);
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: Using 'Log' instead of 'Timber' [LogNotTimber]